OpenKeychain/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,11 @@
     android:installLocation="auto">
 
     <!--
+        General remarks
+        ===============
+        - Last APG 1 version was 10900 (1.0.9 beta 00)
+        - Keychain starting with versionCode 20000!
+
```

Comment:
These have been removed on git master, you somehow rebased your commits in the wrong way.